sophiread/SophireadCLI/CMakeLists.txt

```diff
@@ -1,20 +1,45 @@
+# Include the headers
+include_directories(
+    include
+    ${PROJECT_SOURCE_DIR}/FastSophiread/include
+)
+
 # Configure the commandline application
 set(SRC_FILES
+    src/user_config.cpp
     src/sophiread.cpp
 )
 
-# Include the headers
-include_directories(
-    ${PROJECT_SOURCE_DIR}/SophireadLib/include
-)
-
 # ----------------- CLI APPLICATION ----------------- #
 add_executable(Sophiread
     ${SRC_FILES}
 )
 set_target_properties(Sophiread PROPERTIES VERSION ${PROJECT_VERSION})
-target_link_libraries(Sophiread SophireadLib hdf5 hdf5_cpp OpenMP::OpenMP_CXX)
+target_link_libraries(
+    Sophiread
+    FastSophiread
+    tbb
+    spdlog::spdlog
+    ${HDF5_LIBRARIES}
+)
+
+# ----------------- TESTS ----------------- #
+# UserConfig tests
+add_executable(
+    UserConfigTest
+    tests/test_user_config.cpp
```

Comment:
tests need to be done a little different so they don't get installed. It would take me a minute to dig out the extra attributes that need to be set